components/TextControls/src/private/MDCTextControlColorViewModel.m

```diff
@@ -39,6 +39,13 @@ - (void)setUpColorsWithState:(MDCTextControlState)state {
   UIColor *normalLabelColor = [UIColor darkGrayColor];
   UIColor *leadingAssistiveLabelColor = [UIColor darkGrayColor];
   UIColor *trailingAssistiveLabelColor = [UIColor darkGrayColor];
+  if (@available(iOS 13.0, *)) {
```

Comment:
Please perform compile-time checks for these symbols as well. This code won't compile with Xcode 10.3.

Reply:
Ahhh thank you!